### PR TITLE
Feature/header and footer components

### PR DESCRIPTION
**Pull request by MrOpperman**
Screen Shot 2019-05-07 at 10 18 32 AM
Screen Shot 2019-05-07 at 10 18 26 AM

### CODE REVIEW

File: src/header/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { HeaderContainer } from './style';
+import { ReactComponent as LeadhomeLogo } from '../../assets/leadhome-logo.svg';
```
Comment (seagyn) [inline src/header/index.js]: It would be handy to be able to to pass a logo in so that other people can use it.

File: src/header/style.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import colors from '../colors';
+
+export const HeaderContainer = styled.div`
```
Comment (seagyn) [inline src/header/style.js]: I assume we will worry about improved styling later?

File: src/header/index.js (hunk 1 above)
Comment (MrOpperman) [reply src/header/index.js]: Definitely, going to add that in

File: src/header/style.js (hunk 2 above)
Comment (MrOpperman) [reply src/header/style.js]: Yup.

File: src/header/index.js (hunk 1 above)
Comment (MrOpperman) [reply src/header/index.js]: Leaving the LH one as default so that our projects don't need the svg.

**Pull request merged**